### DISCUSSION TITLE
Citations no longer configuring properly

### DISCUSSION THREAD

**Discussion by rsbyrne** [executablebooks/meta#361] [Q&A] [answered]
Hi there, me again.

For some reason, my citations are not working correctly. I'm trying to change the style to author_year, but instead they are stuck on the 'label' setting:
![image](https://user-images.githubusercontent.com/11673079/118384177-57fc0700-b647-11eb-8a98-4594c9cdebb7.png)

I've set the appropriate key:val pair in my _conf.yaml, but to no avail:
![image](https://user-images.githubusercontent.com/11673079/118384158-37cc4800-b647-11eb-9345-b73ab4c0933f.png)

Of interest might be this error message that pops up when I call `jb build book/`:
![image](https://user-images.githubusercontent.com/11673079/118384143-23884b00-b647-11eb-8a79-13c7c58eef30.png)

For context, here is my current version:
![image](https://user-images.githubusercontent.com/11673079/118384201-8b3e9600-b647-11eb-923d-6c9cfc2bf091.png)

Any help would be so much appreciated!

Cheers,
Rohan

**Comment by choldgraf** [comment, marked as answer]
It sounds like there's a version clash with sohinxcontrib-bibtex...could you list what version you're on? (Or try upgrading it and see if the error persists?)

**Reply by rsbyrne** [reply]
Gosh, you're prompt!!!

Here is my sphinxcontrib-bibtex version:
![image](https://user-images.githubusercontent.com/11673079/118384778-3ea98980-b64c-11eb-8c69-f0aeea5c3642.png)

**Reply by rsbyrne** [reply]
FIXED IT.

You were right, it was a bad sphinxcontrib-bibtex version. I forced `pip` to upgrade it and now the citations are working (in HTML, anyway - the LaTeX PDF ones aren't working still, but that's for another ticket.)

This version works fine:
![image](https://user-images.githubusercontent.com/11673079/118385040-d8723600-b64e-11eb-9d45-981618bfb26c.png)

**Reply by choldgraf** [reply]
wooo 🎉
